backend/test_observer/controllers/artefacts/models.py

```diff
@@ -52,6 +56,8 @@ class TestExecutionDTO(BaseModel):
     c3_link: str | None
     environment: EnvironmentDTO
     status: TestExecutionStatus
+    review_status: list[TestExecutionReviewDecision]
```

Comment:
making `review_status` a set would be more representative and it would be great to add a comment here reminding us why this can have multiple values